src/components/SampleGraphs/sampleGraphs.js

```diff
@@ -49,28 +52,30 @@ MERGE (destination:Place {name: row.destination_reference_place})
 SET destination.countryCode = row.destination_country_code
 
 MERGE (origin)-[eroad:EROAD {road_number: row.road_number}]->(destination)
-SET eroad.distance = toInteger(row.distance), eroad.watercrossing = row.watercrossing`
-        ],
-        algorithms: [
-            {
-                name: "Degree",
-                category: "Centralities",
-                description: "Find the most connected cities on the continent."
-            },
-            {
-                name: "All Pairs Shortest Path",
-                category: "Path Finding",
-                description: "What's the quickest route between the different European cities."
-            }
-        ]
+SET eroad.distance = toInteger(row.distance), eroad.watercrossing = row.watercrossing`,
+      ],
+      algorithms: [
+        {
+          name: "Degree",
+          category: "Centralities",
+          description: "Find the most connected cities on the continent.",
+        },
+        {
+          name: "All Pairs Shortest Path",
+          category: "Path Finding",
+          description:
+            "What's the quickest route between the different European cities.",
+        },
+      ],
     },
-    "Twitter": {
-        name: "Twitter",
-        author: "Mark Needham",
-        authorLink: "https://markhneedham.com/blog/",
-        description: `A dataset containing Twitter followers of the graph community`,
-        queries: [`CREATE CONSTRAINT ON(u:User) ASSERT u.id IS unique`,
-            `CALL apoc.load.json("https://github.com/neo4j-apps/neuler/raw/master/sample-data/twitter/users.json")
+    Twitter: {
+      name: "Twitter",
+      author: "Mark Needham",
+      authorLink: "https://markhneedham.com/blog/",
+      description: `A dataset containing Twitter followers of the graph community`,
+      queries: [
+        `CREATE CONSTRAINT ON(u:User) ASSERT u.id IS unique`,
```

Comment:
constraint syntax?

Reply:
Should be fine as this is 4.0 syntax, while 5.x syntax is https://github.com/neo4j-devtools/neuler/blob/0d83eb767ba81640ff357246a4905b6e733153f6/src/components/SampleGraphs/sampleGraphs.js#L285